AddonTerraformTemplate/main.tf

```diff
@@ -48,6 +48,15 @@ resource "azurerm_monitor_data_collection_endpoint" "dce" {
   kind                = "Linux"
 }
 
+# Create another DCE if the regions don't match
+resource "azurerm_monitor_data_collection_endpoint" "dce_mismatch" {
+  count               = local.dce_region_mismatch ? 1 : 0
```

Comment:
Add a variable here to only create this DCE if the customer wants to use private link?

Reply:
done